data/cloud_services.yaml

```diff
@@ -20,12 +20,14 @@ sources:
     paths:
     - '%%users.appdata%%\Dropbox\*.db*'
     - '%%users.localappdata%%\Dropbox\*.db*'
+    - '%%users.homedir%%/.dropbox/instance1/sync_history.db'
```

Comment:
you'll need to add a separate source type for this

Reply:
My late night Ctrl+C -V failed me.  I've updated with the correct value and local unit tests appear to work.